Misc/NEWS.d/next/Core and Builtins/2019-02-13-11-50-30.bpo-35982.LZubLj.rst

```diff
@@ -0,0 +1 @@
+Add unittest for os.renames(). Contributed by Stéphane Wirtel.
```

Comment:
Can you add markup format to `os.renames()`?

Reply:
sure, done.